clients/admin-ui/cypress/e2e/config-wizard-plus.cy.ts

```diff
@@ -233,14 +233,13 @@ describe("Config wizard with plus settings", () => {
       cy.getByTestId("add-systems");
     });
 
-    // TODO: Update Cypress test to reflect the nav bar 2.0
-    it.skip("Resets the flow when it is completed", () => {
+    it("Resets the flow when it is completed", () => {
       goToDataFlowScanner();
       cy.wait("@putScanResults");
       cy.getByTestId("scan-results");
       cy.getByTestId("register-btn").click();
-      cy.getByTestId("nav-link-Add Systems").click();
-      cy.getByTestId("setup");
+      cy.visit("/add-systems");
+      cy.getByTestId("add-systems");
```

Comment:
@allisonking I'm not 100% sure what you _meant_ to test by the "reset the flow when it's completed" based on your comment on the other PR, but I think it's the stateful-ness of that flow and that the `add-systems` page should be displayed. So I think this flow (navigate to add systems again, assert that it shows that test id) is covering what you wanted here